src/routes/states.js

```diff
@@ -269,7 +269,7 @@ module.exports = (errorHandler, stateService) => {
   /**
    * @apiGroup geofence_states
    * @api {get} /states/layers/national NationalLayer
-   * @apiName StatesNationalLayer
+   * @apiName StatesNationalLayer2
```

Comment:
Is this name intentioned? Or the 2 was a typo?

Reply:
totally a typo (and very strange). Fixed it.